gradle.properties

```diff
@@ -7,10 +7,10 @@ commonsLang3Version=3.12.0
 commonsTextVersion=1.10.0
 
 grgitGradleVersion=5.0.0
-graphqlJavaVersion=18.1
+graphqlJavaVersion=20.0
 
 httpclientVersion=4.5.14
 httpcoreVersion=4.4.16
 
-jacksonVersion=2.13.3
-jsonVersion=20220320
+jacksonVersion=2.14.1
```

Comment:
Choosing 2.14.1 to align with sever build (for easier property reuse at the moment).

Reply:
Fair enough. Tangentially related: I've been thinking about adding a `commonsTextVersion` to the server `gradle.properties`. This library is only used in a couple EHR modules right now but 1) might as well sync them to the same version and 2) might actually be time to pull this into api since some commons lang methods we use are deprecated in favor of commons text. Not saying we need to do anything in this PR...

Reply:
Oh, cool, just saw your commit over in server.

Reply:
I added `commonsTextVersion` over on server with a comment about its being used for gradle plugins. We can remove that comment if it's actually used elsewhere, but I'll leave it as is for now.

Reply:
Sounds good... thanks!